core/vm/evm.go

```diff
@@ -214,8 +215,14 @@ func (evm *EVM) Call(caller ContractRef, addr common.Address, input []byte, gas
 			}
 			return nil, gas, nil
 		}
+
 		evm.StateDB.CreateAccount(addr)
 	}
+	if evm.Accesses != nil && value.Sign() != 0 {
```

Comment:
possibly a good target for replacement with `IsCancun`

Reply:
will do that in a subsequent PR as we've agreed.